styles/header.module.scss

```diff
@@ -0,0 +1,62 @@
+$primary: #00629B;
+$secondary: #023D54;
+$primary-background: #fff;
+$sm-screen: 480px;
+$md-screen: 768px;
+
+.header {
+    background-color: $primary-background;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 10px 80px;
+    .logo {
+      display: flex;
+      align-items: center;
+      margin-right: 40px;
+    }
+    .nav {
+      display: inline-block;
+      ul {
+        display: flex;
+        gap: 36px;
+        list-style-type: none;
+        li {
+          display: inline-block;
+          a {
+            color: $primary;
+            font-size: 16px;
+            font-weight: bold;
+            
+          }
+        }
+      }
+    }
+    .menuIcon {
+      display: none;
+      color: $primary;
+      align-items: center;
+      cursor: pointer;
+      
+    }
+  }
+  
+  @media screen and (max-width: $md-screen) {
+    .header {
+      margin: 10px 24px;
+      .logo {
+        img {
+          width: 80px;
+          height: 80px
+        }
+      }
+      .nav {
```

Comment:
Se for mudar aquela div para nav, lembra de mudar esse cara aqui para nav em vez de .nav